src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -92,7 +93,16 @@ public function getTranslationNodeVisitor(): TranslationNodeVisitor
         return $this->translationNodeVisitor ?: $this->translationNodeVisitor = new TranslationNodeVisitor();
     }
 
-    public function trans(string $message, array $arguments = [], string $domain = null, string $locale = null, int $count = null): string
+    /**
+     * @param string|Translatable $message   The message id (may also be an object that can be cast to string)
+     * @param array               $arguments An array of parameters for the message
```

Comment:
I would remove all parameters where we don't need the description (including the @return one). I would only keep `@param string|Translatable $message`.

Reply:
Updated